engine/src/gfx/cockpit.cpp

```diff
@@ -2008,7 +2008,7 @@ void GameCockpit::Draw() {
             const float deadband = configuration()->joystick.mouse_deadband;
             const int reverse_spr = configuration()->joystick.reverse_mouse_spr;
             const string blah = configuration()->joystick.mouse_crosshair;
-            VSSprite MouseVSSprite(blah.c_str(), BILINEAR, GFXTRUE);
+            static VSSprite MouseVSSprite(blah.c_str(), BILINEAR, GFXTRUE);
```

Comment:
This is what fixed the performance issue, I believe. When I made this variable non-static, all of a sudden we had to construct a new `VSSprite` every time this code was hit. Very computationally expensive.

Reply:
I guess the lesson here is to leave computation-heavy and I/O-heavy object variables as `static`, even if their value depends on another variable that is now `const` and not `static`. At least, wherever that is feasible.

Reply:
@BenjamenMeyer done.